web/content/docs/tools/meshing-submeshes/extract-entire-surface/index.pandoc

```diff
@@ -0,0 +1,49 @@
++++
+date = "2020-01-17T5:56:57+01:00"
+title = "Extract Entire Surface"
```

Comment:
I suggest to properly name what is now called "surface"---a 2d entitiy---a boundary, which could be 0d to 2d (for given 3d spaces).

Reply:
Will rename the tool and change it in the docu.